# Fast and dirty patch to change the default useragent. Proper implementation incoming
--- a/utils/useragent.c
+++ b/utils/useragent.c	
@@ -63,7 +63,7 @@
                  netsurf_version_minor,
                  sysname);
 
-        core_user_agent_string = ua_string;
+	 core_user_agent_string = "Mozilla/5.0 (Windows NT 10.0; rv:68.0) Gecko/20100101 Firefox/68.0";
 
         NSLOG(netsurf, INFO, "Built user agent \"%s\"",
               core_user_agent_string);
